src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -61,8 +72,11 @@
 
     @Test
     public void getStatisticAboutBanana() {
+        try {
         workWithFile.getStatistic("banana.csv", BANANA_RESULT_FILE);
-
+        } catch (FileNotFoundException e) {
+            System.out.println("Can't write data to file");
```

Comment:
The error message 'Can't write data to file' is not very informative. Consider using a more descriptive message that includes the exception details.